spec/classes/openvmtools_init_spec.rb

```diff
@@ -143,20 +143,19 @@
     }
   end
 
-  context 'on a supported Debian 14 os, vmware platform, default parameters' do
+  context 'on a supported Ubuntu 14.04 os, vmware platform, default parameters' do
     let(:params) { {} }
     let :facts do
       {
         operatingsystem:           'Ubuntu',
-        operatingsystemmajrelease: '14',
+        operatingsystemmajrelease: '14.04',
         operatingsystemrelease:    '14.04',
         os:                        {
           family:  'Debian',
           name:    'Ubuntu',
           release: {
             full:  '14.04',
-            major: '14',
```

Comment:
You can verify at https://github.com/camptocamp/facterdb/blob/master/facts/3.14/ubuntu-14.04-x86_64.facts#L336-L339